packages/plop/package.json

```
@@ -0,0 +1,60 @@
+{
+  "name": "plop",
+  "version": "3.0.5",
+  "description": "Micro-generator framework that makes it easy for an entire team to create files with a level of uniformity",
+  "main": "./src/plop.js",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/plopjs/plop.git",
+    "directory": "packages/plop"
+  },
+  "keywords": [
+    "generator",
+    "scaffolding",
+    "yeoman",
+    "make",
+    "build",
+    "generate",
+    "gen",
+    "plop"
+  ],
+  "author": "Andrew Worcester <andrew@amwmedia.com> (http://amwmedia.com)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/plopjs/plop/issues"
+  },
+  "scripts": {
+    "test": "npm run test:instrument && npm run vitest && nyc report",
+    "test:instrument": "nyc instrument ./bin ./instrumented/bin && nyc instrument ./src ./instrumented/src && cp package.json ./instrumented",
+    "vitest": "vitest run",
+    "prepublishOnly": "node ./scripts/prepublishOnly.js",
+    "postpublish": "node ./scripts/postpublish.js"
+  },
+  "devDependencies": {
+    "cli-testing-library": "^1.0.1",
+    "inquirer-directory": "^2.2.0",
+    "nyc": "^15.1.0",
+    "plop-pack-fancy-comments": "^0.2.0",
+    "queue-microtask": "^1.2.3",
+    "vitest": "^0.5.5"
+  },
+  "homepage": "https://plopjs.com",
+  "dependencies": {
+    "@types/liftoff": "^4.0.0",
+    "chalk": "^5.0.0",
+    "interpret": "^2.2.0",
+    "liftoff": "^4.0.0",
+    "minimist": "^1.2.5",
+    "node-plop": "*",
```

This will 100% break when we release a, say, v2 of `node-plop`. 

We either need to lock this to:

```suggestion
    "node-plop": "~0.30.0",
```

Or, if we decide to bump to `1.0.0`, like I'm hoping we can:
```suggestion
    "node-plop": "^1.0.0",
```

---

Agreed. Also, caret works for both so we can use it to select nonbreaking changes either way.

https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004